drivers/input/CMakeLists.txt

```diff
@@ -1,5 +1,7 @@
 # SPDX-License-Identifier: Apache-2.0
 
+add_subdirectory_ifdef(CONFIG_INPUT_PINNACLE pinnacle)
```

Comment:
You sure about the `pinnacle` name? That's the ASIC name right? So it makes sense to me, just figure I'd ask before we commit to it.

Reply:
@fabiobaltieri, the ASIC's name is 1CA027 and this is their [Gen2/3 ASIC ](https://www.cirque.com/gen2gen3-asic-details)which has the Pinnacle codename. I decided to use the codename instead of 1CA027 because it starts from a digit which creates problem by using in it macros and function names.

This ASIC is used in all original GlidePoint modules which are currently available on the market. I noticed that the product page of [GlidePoint circle trackpad modules](https://www.cirque.com/glidepoint-circle-trackpads) was updated several months ago and it doesn't mention old modules anymore. Instead, it shows modules which look slightly different from all currently available ones and it mentions that they are based on their Gen6 ASIC. I would speculate that this is a new generation of modules.